e2e/tests/testOdooIntegration.spec.ts

```diff
@@ -36,7 +36,7 @@ test('Patient with lab order becomes customer in Odoo', async ({ page }) => {
   await expect(quotation?.includes("Quotation")).toBeTruthy();
 });
 
-test('Editing patient details with a synced lab order edits the corresponding customer details in Odoo', async ({ page }) => {
+test('Editing details of OpenMRS patient with a synced lab order edits the corresponding Odoo customer details', async ({ page }) => {
```

Comment:
As discussed on Slack, we should choose between full English of abbreviated/computer English:

>Editing the details of an OpenMRS patient with a synced lab order edits the corresponding Odoo customer details.

vs

>Editing details of OpenMRS patient with synced lab order edits corresponding Odoo customer details

Let's make that choice and apply it everywhere, ok?

Reply:
Ok then please re-read all of them and apply the full English writing, including the end of sentence dot :-)